crates/hyperqueue/src/dashboard/ui/screens/worker/cpu_util_table.rs

```diff
@@ -0,0 +1,58 @@
+use std::cmp;
+use tui::layout::{Constraint, Rect};
+use tui::widgets::{Cell, Row, Table};
+
+use crate::dashboard::ui::styles;
+use crate::dashboard::ui::terminal::DashboardFrame;
+use crate::dashboard::ui::widgets::progressbar::{
+    get_progress_bar_color, render_progress_bar_at, ProgressPrintStyle,
+};
+
+const CPU_METER_LEN: u8 = 18;
+const PADDING: u8 = 4;
+
+pub fn render_cpu_util_table(
+    usages: &[f32],
+    rect: Rect,
+    frame: &mut DashboardFrame,
+    constraints: &[Constraint],
+) {
+    let rows: Vec<Row> = usages
```

Comment:
Could you please try to prepend the CPU id (it's virtual, but anyway, just its index within the vec) before the progress bars?